DistValetDriver.php

```diff
@@ -23,7 +27,7 @@ public function serves($sitePath, $siteName, $uri)
      * @param  string  $uri
      * @return string|false
      */
-    public function isStaticFile($sitePath, $siteName, $uri)
+    public function isStaticFile(string $sitePath, string $siteName, string $uri)/*: string|false */
```

Comment:
Sorry it took me so long. Should this return statement be a comment?

Reply:
yes, in Valet 4 the return-type is commented out by default in order to not crash on older PHP versions. At some point when older versions fall away this won't be an issue any longer, but this is consistent with Valet internals for now.

Reply:
Thank you for the explanation!